src/gitHubCopilotForAzure.ts

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { callWithTelemetryAndErrorHandling, IActionContext, openUrl } from "@microsoft/vscode-azext-utils";
+import { commands, Extension, ExtensionContext, extensions, window } from "vscode";
+import { localize } from "./utils/localize";
+
+const ghcpExtensionId = 'github.copilot';
+const ghcpChatExtensionId = 'github.copilot-chat';
+const ghcp4aExtensionId = 'ms-azuretools.vscode-azure-github-copilot';
+const ghcp4aLearnPage = 'https://aka.ms/GetGitHubCopilotForAzure';
+const dontShowKey = 'ghcp4a/dontShow';
+
+/**
+ * Conditionally show an install toast for the GitHub Copilot for Azure extension
+ */
+export function gitHubCopilotForAzureToast({ globalState }: ExtensionContext): void {
+    void callWithTelemetryAndErrorHandling('ghcp4aToast', async (context: IActionContext) => {
```

Comment:
Since this telemetry event is getting sent for all users on every activation, we should do `isActivationEvent = 'true'`

Reply:
For this: `Otherwise we should suppress it anytime the notification is not shown`

Should I test for early return before calling `callWithTelemetryAndErrorHandling` then, or is the way I updated it now fine?